### PR TITLE
Introduces withdrawal screen

### LINKED ISSUE #1380: I want to transfer tokens back to the main account
## User Story
As Chris I want to transfer tokens back to the main account so that I can use them in eg. MetaMask.

After discussion of https://github.com/raiden-network/light-client/issues/1301 and https://github.com/raiden-network/light-client/issues/1314

## Acceptance criteria
- I only have got the option to withdraw all tokens

## First Discussion
@taleldayekh @nephix and @christianbrb 

### User flow:
1. Channel is getting closed by the partner
2. Channel is getting settled by the partner
3. Tokens out of channel are transferred from channel to the Raiden account (subkey)
4. Transfer ETH to Raiden Account to pay for gas -> signed in MetaMask (#1365) 
5. Transfer tokens from Raiden Account to Main (ETH) Account -> signed in dApp (background)

### Proposed solution:
- Having a button on the general screen for "token balances"
- List of tokens are shown and each token balance can be withdrawn
- Tell the user before settling, that the token balance can be withdrawn on the Account Screen (new naming for General screen)

### Wireframe

wireframes__withdrawal

### Design / Mockups
https://xd.adobe.com/view/c95c87f5-d595-4ef8-7b9e-5b57e4e51941-bf6d/

### Question:
- @andrevmatos How does the subkey works for the UDC (SVT tokens)?

### PR DESCRIPTION
**Pull request by kelsos**
Fixes #1380

**Short description**
Introduces withdraw for tokens from the Raiden Account

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.  Go to the `Account` page
2. Press `Withdrawal`
3. Click on a token to withdraw

> You can go to goerli on etherscan and mint some tokens for the subkey account, if you don't want to go through the opening and closing the channel.

### CODE REVIEW

**Comment by kelsos** [comment]
@christianbrb There are two things remaining to consider this issue done. 

- Evaluate what happens when the user opens the `Withdraw` screen directly after connecting
- Write some tests for the service part, based on the outcome of the evaluation.

The problem is that if the channels settle, not token is returned anymore so my initial thought of not fetching everything doesn't work. Fetching every token and the balances of tokens on the subkey is also a bit problematic, as it takes a lot of time to load and I usually observe errors in the console due to failed requests.

**Comment by christianbrb** [comment]
@kelsos Thanks for the update :)

I would just store the token locally on open channel. You can delete the local storage once withdraw has been successful.

**Comment by kelsos** [comment]
@christianbrb that would be a solution, but I am not that happy with it, give me some time to check and evaluate solutions. Caching will potentially save us some effort here (but I am talking about the token caching)

File: raiden-dapp/src/components/account/Withdrawal.vue [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    background-color: transparent;
+
+    &__balance {
+      margin-right: 12px;
+    }
+
+    &__address {
+      ::v-deep {
+        .address__label {
+          color: #ffffffb3 !important;
```
Comment (nephix) [inline raiden-dapp/src/components/account/Withdrawal.vue:197]:
```suggestion
          color: rgba($color-white, 0.7) !important;
```

But would also need `@import '@/scss/colors';`

Comment (kelsos) [reply raiden-dapp/src/components/account/Withdrawal.vue:197]: ok let me check that and also fix the linting issue that I didn't see locally :disappointed:

**Comment by nephix** [comment]
Tests were successful. After settling a channel I had the balance display on the withdrawal screen and was able to withdraw it to my main account

**Pull request merged**